sale_order_import_invoice2data/wizard/sale_order_import_pdf.py

```diff
@@ -0,0 +1,178 @@
+        precision = self.env['decimal.precision'].precision_get('Product UoS')
+        parsed_order = {
+            'partner': {
+                'vat': invoice2data_res.get('vat'),
+                'name': invoice2data_res.get('partner_name'),
+                'email': invoice2data_res.get('partner_email'),
+                'website': invoice2data_res.get('partner_website'),
+                'siren': invoice2data_res.get('siren'),
+            },
+            'currency': {
+                'iso': invoice2data_res.get('currency'),
+            },
+            'amount_total': invoice2data_res.get('amount'),
+            'saleorder_number': invoice2data_res.get('invoice_number'),
+            'date': invoice2data_res.get('date'),
+            'date_due': invoice2data_res.get('date_due'),
+            'date_start': invoice2data_res.get('date_start'),
+            'date_end': invoice2data_res.get('date_end'),
+            'description': invoice2data_res.get('description'),
+        }
+        if 'amount_untaxed' in invoice2data_res:
+            parsed_order['amount_untaxed'] = invoice2data_res['amount_untaxed']
+        if 'lines' in invoice2data_res:
+            parsed_order['lines'] = []
+            for i, line in enumerate(invoice2data_res['lines']):
+                vals = {
+                    # TODO: bdio.compare_lines() expects
+                    # this way,
+                    'product': {
+                        'desc': line.get('desc'),
+                        'code': line.get('code'),
+                        'ean13': line.get('ean13'),
+                    },
+                    # sale_order_import expects this way,
+                    'desc': line.get('desc'),
+                    'code': line.get('code'),
+                    'ean13': line.get('ean13'),
+                    # --------------
+                    'taxes': line.get('taxes'),
+                    'unit_price': line.get('unit_price'),
+                    'price': line.get('price'),
+                }
+                # Quantity
+                try:
+                    qty = float(line.get('qty'))
+                except:
+                    raise UserError(_(
+                        "Error on PDF order line %d: The quantity should "
+                        "use dot as decimal separator and shouldn't have any "
+                        "thousand separator") % i)
+                if float_compare(qty, 0, precision_digits=precision) != 1:
+                    raise UserError(_(
+                        "Error on PDF order line %d: the quantity should "
+                        "be strictly positive") % i)
+                vals['qty'] = qty
+                parsed_order['lines'].append(vals)
+        if 'amount_tax' in invoice2data_res:
+            parsed_order['amount_tax'] = invoice2data_res['amount_tax']
+        return parsed_order
+
+    @api.model
+    def create_order(self, parsed_order, price_source):
```

Comment:
what's this override good for? Looks suspiciously like you just copied https://github.com/OCA/edi/blob/9.0/sale_order_import/wizard/sale_order_import.py#L212